### ISSUE TITLE
TestID and Accessibility Label missing on iOS from Contact on Settings Page

### ISSUE THREAD

**Issue by nodlesh** [bcgov/bc-wallet-mobile#997]
On the Settings page where you would select Contacts to get to the Contacts List. 
There is no TestID like `com.ariesbifold:id/Contacts` on Contacts.
There is no useful Accessibility label as well.  
 
![Screenshot 2023-03-23 at 5.54.51 PM.png](https://images.zenhubusercontent.com/5e3864b3b64bc2429a017886/8680ab8e-273d-4eb2-8714-3d9414f7431e)

### Environment
BC Wallet build 756
Tested on iOS 15

### Workaround
Can only run on Android

### Severity
High

### Priority
High

### Test Impact
All Remove Contact tests
